src/etc/rc.subr.d/recover

```diff
@@ -199,6 +200,7 @@ variables_order = "GPCS"
 file_uploads = On
 upload_tmp_dir = /tmp
 upload_max_filesize = 200M
+max_file_uploads = 1
```

Comment:
for some plugins this might be challenging, in our code this shouldn't matter.

Reply:
I couldn't find a `$_FILES` use in plugins. The default is 20, assuming 200 MB per file I'd say more than 5 is already pretty exhaustive

Reply:
@muratbalaban43 could you check this? Are you using file uploads extensively? Maybe in Phalcon using `getUploadedFiles()`

Reply:
@muratbalaban43 going with 5 for now, probably not before 24.7

Reply:
@fichtner thanks for the heads-up. looking into this now.